packages/wonder-blocks-layout/components/media-layout.js

```diff
@@ -52,104 +67,72 @@ type State = {|
 // fall back to this state.
 const DEFAULT_SIZE = "large";
 
+type CombinedProps = {|...Props, ...Context|};
+
 /**
  * `MediaLayout` is responsible for changing the rendering of contents at
  * differently sized viewports.  `MediaLayoutContext.Provider` can be used
  * to specify different breakpoint configurations.  By default it uses
  * `MEDIA_DEFAULT_SPEC`.  See media-layout-context.js for additiional options.
  */
-export default class MediaLayout extends React.Component<Props, State> {
-    watchHandlers: ?{
-        [query: string]: any,
-    };
-
-    static WATCHERS: {
-        [query: string]: any,
-    } = {};
+class MediaLayoutInternal extends React.Component<CombinedProps, State> {
+    cleanupThunks: Array<() => void>;
```

Comment:
Can you add a comment explaining this variable? It's not super obvious on first glance.